### DISCUSSION TITLE
springboot （2.5.0）整合 Simbot （版本：2.4.0）

### DISCUSSION THREAD

**Discussion by 930893138** [ForteScarlet/simpler-robot#457] [不要在这里发帖子了]
配置一切按照2.0的文档书写，但是启动报错创建这个simbotContext异常，（本人尝试过只使用这两个结合，排除自己的其他代码的影响，但是还是报错）
错误全称：```
Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'simbotContext' defined in love.forte.simbot.spring.autoconfigure.SimbotAppConfiguration: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [love.forte.simbot.core.SimbotContext]: Factory method 'simbotApp' threw exception; nested exception is love.forte.common.ioc.exception.NoSuchDependException: love.forte.simbot.api.sender.SenderFactory
```

**Comment by ForteScarlet** [comment]
此异常大概率为缺失部分依赖或启动方式错误

可以试着参考 https://github.com/simple-robot/simbot-mirai-spring-boot-demo 的配置
